app/src/Home.tsx

```diff
@@ -4,69 +4,21 @@ import AddJoke from "./AddJoke";
 import BackButton from "./BackButton";
 import NextButton from "./NextButton";
 
-interface Joke {
-  text: string;
-  complete: boolean;
-}
-
-interface HomeProps {
-  onClick: () => void;
-}
-
-export default function Home({ onClick }: HomeProps): JSX.Element {
+export default function Home(): JSX.Element {
   const [count, setCount] = useState<number>(0);
-  const newJokeToLaugh: Joke[] = [
-    {
-      text: "What do you call a yam with a broom ? A sweep potato.",
-      complete: false,
-    },
-    {
-      text: "Why did the ice cream van break down? Because there was a rocky road!",
-      complete: true,
-    },
-  ];
-  const [haJokes, setHaJokes] = useState<Joke[]>(newJokeToLaugh);
-
-  const toggleJoke = (selectedJoke: Joke) => {
-    const newJokes = haJokes.map((joke) => {
-      if (joke === selectedJoke) {
-        return {
-          ...joke,
-          complete: !joke.complete,
-        };
-      }
-      return joke;
-    });
-    setHaJokes(newJokes);
-  };
 
   return (
     <>
       <AppStyle>
         <NextButton to={"/joke-randomiser"} />
         <BackButton to={"/joke-fetch"} />
         <StyledTitle>Giggle Click</StyledTitle>
-        {/* <ul>
```

Comment:
Thank you for deleting this commented out code of checkbox joke and line-through, way more cleaner now 🧹 🧼 🧤